extras/translations/de.po

```diff
@@ -531,7 +535,9 @@ msgstr "Verschlüsselungsschlüssel konnte nicht erstellt werden."
 
 #: tomb:Create:forge_key:1492
 msgid "Commanded to forge key ::1 key:: with cipher algorithm ::2 algorithm::"
-msgstr "Der Schlüssel ::1 key:: wird mit dem Verschlüsselungsalgorithmus ::2 algorithm:: geschmiedet."
+msgstr ""
+"Der Schlüssel ::1 key:: wird mit dem Verschlüsselungsalgorithmus ::2 "
+"algorithm:: gefälscht"
```

Comment:
Well, this isn't a wrong translation for forge, but it is wrong in this case

Reply:
what you mean? the swedish patch seems fine

Reply:
In this case forge was translated as to fake something. Not in its meaning to create something. And this particular german word doesn't have multiple meanings.
Translated back it kinda sounds like: the key is faked with the algorithm. Not really convincing to read :P